src/util_exec.cpp

```diff
@@ -576,10 +576,10 @@ Util::runMtcpRestore(int is32bitElf,
   size_t argvSizeDiff = originalArgvEnvSize - newArgvEnvSize;
   if (argvSizeDiff > 0) {
     dummyEnviron = (char *)malloc(argvSizeDiff);
-    memset(dummyEnviron, '0', argvSizeDiff - 1);
+    memset(dummyEnviron, 0, argvSizeDiff - 1);
```

Comment:
Here we do want the character `'0'` instead of nul byte. The reason is that we are trying to create a big env var that will be of the form `DUMMY_ENV=00000000000` so that it takes up `argvSizeDiff-1` bytes in length..